lib/covenants/rules.js

```diff
@@ -416,12 +417,17 @@ rules.isLockedUp = function isLockedUp(nameHash, height, network) {
   if (height < network.names.claimPeriod)
     return false;
 
-  const item = reserved.get(nameHash);
+  const item = locked.get(nameHash);
 
   if (!item)
     return false;
 
-  if (item.root || item.top100 || item.custom)
+  // ICANN Names are always LOCKED.
+  if (item.root)
+    return true;
+
+  // Alexa names will expire after 4 years.
```

Comment:
Should it be 8 years?    
```suggestion
  // Alexa names will expire after 8 years.
```

At least _lib/protocol/networks.js_ has the following:  

```javascript
  /**
   * The time window which the
   * names will be locked before release or hard fork.
   * @const {Number}
   */

  alexaLockupPeriod: (8 * 365) * main.pow.blocksPerDay,
```